lib/autoit.js

```
@@ -43,7 +43,11 @@ function AutoItObject() {
         if (this._interface != null) {
             try {
                 //this._interface[functionName].apply(null, args);
-                eval("this._interface." + functionName + "(\"" + args.map(addslashes).join("\", \"") + "\")");
+                var evaluate = eval;
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)